packages/content/src/Infrastructure/Symfony/Command/DebugContentCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Content\Infrastructure\Symfony\Command;
+
+use Sulu\Content\Application\ContentResolver\Resolver\ResolverInterface;
+use Sulu\Content\Application\PropertyResolver\Resolver\PropertyResolverInterface;
+use Sulu\Content\Application\ResourceLoader\Loader\CachedResourceLoader;
+use Sulu\Content\Application\ResourceLoader\Loader\ResourceLoaderInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand('sulu:debug:content', description: 'Debugging the content services to see if they are correctly registered')]
+final class DebugContentCommand extends Command
```

Comment:
We prefix all new commands with the bundle name so if this lives in SuluContentbundle it should be `sulu:content:debug` naming convention is `<namespace>:<bundle>:<commandname>`.

While we provide BC for the command name arguments, options we not provide any bc for the command class itself so we mark the command class as `@internal` as no other need. This adds as easier possibility to add more debug informations in future:

```suggestion
/**
 * @internal The command can be used and console command call is under bc promise but no bc promise is given for this PHP class itself
 * as it may will have additional dependencies in future.
 */
#[AsCommand('sulu:content:debug', description: 'Debugging the content services to see if they are correctly registered')]
final class DebugContentCommand extends Command
```

Reply:
Sounds good. Should I also create a pr for all the other commands to add aliases to them so that we can name them the same way?

Reply:
for existing we currently keep the old names.